src/app/chart/donut/examples/donut-example.component.ts

```diff
@@ -54,5 +55,15 @@ export class DonutExampleComponent implements OnInit {
       return 'Pets';
     };
     this.config2.chartHeight = 120;
+
+    Observable
+      .timer(0, 1000)
+      .map(() => Math.floor(Math.random() * 5) + 1)
+      .subscribe(val => this.data[0][1] = val);
+
+    Observable
+    .timer(0, 5000)
+    .map(() => Math.floor(Math.random() * 100) + 100)
+    .subscribe(val => this.config2.chartHeight = val);
```

Comment:
This is very cool, but can we add this as a separate example? For example, have a "Basic" and "Dynamic" tabs similar to other component examples.

Sometimes my tooltip disappears as I'm attempting to hover and the data changes under me.

Reply:
As an example, we could have examples named donut-basic-example.component.ts and donut-dynamic-example.component.ts...